dataprofiler/tests/profilers/test_unstructured_labeler_profile.py

```diff
@@ -121,6 +121,11 @@ def test_profile(self, processor_class_mock, model_class_mock):
                 'true_char_level': defaultdict(int, {'UNKNOWN': 1}),
                 'word_level': defaultdict(int)
             },
+            entity_percentages={
+                'postprocess_char_level': defaultdict(int, {'UNKNOWN': 1.0}),
+                'true_char_level': defaultdict(int, {'UNKNOWN': 1.0}),
+                'word_level': defaultdict(int)
```

Comment:
Can we add more entities to this test so that there are several values for each dict?